src/test/java/com/github/tomakehurst/wiremock/extension/responsetemplating/ResponseTemplateTransformerTest.java

```diff
@@ -1057,11 +1057,11 @@ public void joinWithObjectBody() {
                 + "  { \"id\": 3, \"name\": \"Three\" }\n"
                 + "]\n"
                 + "{{/parseJson}}"
-                + "[{{#join ',' myThings as |item|}}"
+                + "[{{#arrayJoin ',' myThings as |item|}}"
```

Comment:
Can we use the `prefix` and `suffix` parameters with the block style `arrayJoin` ?

Reply:
Yes. However, I found the same problem, the order of the parameters are not intuitive(IMHO):

```{{#arrayJoin  ',' myThings prefix='[' suffix=']' as |item|}}
{
name:  "{{item.name}}"
}
{{/arraayJoin}}
```

On the other hand, the same thing can be achieved without prefix/suffix
```
[
{{#arrayJoin  ',' myThings as |item|}}
{
name:  "{{item.name}}"
}
{{/arraayJoin}}
]
```